coderd/notifications/manager.go

```diff
@@ -141,6 +141,7 @@ func (m *Manager) Run(ctx context.Context) {
 	m.log.Info(ctx, "started")
 
 	m.runOnce.Do(func() {
+		m.notifier = newNotifier(ctx, m.cfg, uuid.New(), m.log, m.store, m.handlers, m.helpers, m.metrics, m.clock)
```

Comment:
Let's add a comment still why we're initializing this here so it doesn't move by accident 👍🏻

Reply:
I think @defelmnq is on PTO, @mafredri would you mind adding the comment so we can merge it?

PS. not sure what happened that 9 CI jobs failed here...